README.md

```diff
@@ -113,7 +185,7 @@ tox -e behave -D reuse_container=container_name
 
 ## Building
 
-The packaging for the UA client package (ubuntu-advantage-tools) is
```

Comment:
Thanks for updating this. Why are you just mentioning the -pro package?

Reply:
I also added some context above in the intro
```
Ubuntu Pro images on AWS and Azure Ubuntu Pro 
images will also contain `ubuntu-advantage-pro` which automates machine attach  
on custom AWS and Azure images. 
```